components/gsl/Makefile

```diff
@@ -9,36 +9,39 @@
 #
 
 #
-# Copyright (c) 2013 Louis M. Picciano. All rights reserved.
+# Copyright (c) 2013-2016 Louis M. Picciano. All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include $(WS_TOP)/make-rules/shared-macros.mk
```

Comment:
BTW the component actually redeclares the FMRI in the manifest as developer/gsl instead of library/math/gsl.
Same applies to the classification.